ultralytics/nn/autobackend.py

```diff
@@ -666,6 +669,6 @@ def _model_type(p="path/to/model.pt"):
             from urllib.parse import urlsplit
 
             url = urlsplit(p)
-            triton = bool(url.netloc) and bool(url.path) and url.scheme in {"http", "grpc"}
+            triton = (bool(url.netloc) and bool(url.path) and url.scheme in {"http", "grpc"}) or p.startswith("dns:///")
```

Comment:
If `dns` is a URL scheme why is it treated specially here instead of being added to the set of valid URL schemes that we already have in `{"http", "grpc"}`?

Reply:
you are right, will check it.